meteor-app/.meteor/packages

```diff
@@ -37,3 +37,4 @@ random@1.1.0
 crosswalk@1.7.1
 email
 grove:call-async
+momentjs:moment
```

Comment:
It's better to use npm modules if there is a choice, as meteor packages may use outdated versions of the package